pkg/asset/manifests/infrastructure_test.go

```diff
@@ -11,12 +11,23 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/ipnet"
 	"github.com/openshift/installer/pkg/types"
 	awstypes "github.com/openshift/installer/pkg/types/aws"
 	azuretypes "github.com/openshift/installer/pkg/types/azure"
 	"github.com/openshift/installer/pkg/types/dns"
 	gcptypes "github.com/openshift/installer/pkg/types/gcp"
 	nonetypes "github.com/openshift/installer/pkg/types/none"
+	vspheretypes "github.com/openshift/installer/pkg/types/vsphere"
+)
+
+const (
+	defaultMachineNetwork = "10.0.0.0/16"
+	ipv6MachineNetwork    = "fe80::/10"
```

Comment:
I just noticed, this is actually not a good example. There are multiple places where we specifically forbid using link-local IPv6 range. It may pass the unit test here because those are designed to be simple, but it can stop working any day.

The bad ones are also IPv4-mapped subnets (`::FFFF:0:0/96`) and multicast (`ff00::/8`). I think a reasonable choice can be documentation subnet `2001:db8::/32` because it's not meant to be treated in any special way

Reply:
Thanks Mat. I have a lot to learn about networking & particularly ipv6, so I appreciate your guidance. I will update the unit tests in a bit.

Reply:
fixed and squashed commits